### PR TITLE
yaml: no quotation; src: same quotation; issues: add auto auther assign #78

### PR DESCRIPTION
**Pull request by seunggabi**
#78

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
+.idea
```
Comment (toshimaru) [inline .gitignore:1]: Please ignore this in your computer only. 💻 

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: 'Auto Author Assign'
+name: Auto Author Assign
```
Comment (toshimaru) [inline .github/workflows/ci.yml:1]: These yaml changes are cosmetic. I don't want to take the cosmetic changes.

Could you revert these changes?

Comment (seunggabi) [reply .github/workflows/ci.yml:1]: @toshimaru 
ok I rollback yaml files.

but, I recommend to read this~
https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml

yaml syntax no quotation; because string is default.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,8 +2,10 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [1.5.0](https://github.com/toshimaru/auto-author-assign/compare/v1.4.0...v1.5.0) (2022-03-28)
+### [1.5.1](https://github.com/toshimaru/auto-author-assign/compare/v1.5.0...v1.5.1) (2022-07-22)
```
Comment (seunggabi) [inline CHANGELOG.md:5]: please fix this version @toshimaru

File: .github/workflows/ci.yml (hunk 2 above)
Comment (toshimaru) [reply .github/workflows/ci.yml]: Thanks for the information. :)

Okay, I'm gonna take this change in another PR. 👍

**Review by toshimaru — APPROVED** [review]

**Comment by toshimaru** [comment]
just released v1.6.0 as a pre-version.

https://github.com/toshimaru/auto-author-assign/releases/tag/v1.6.0

File: .github/workflows/ci.yml (hunk 2 above)
Comment (toshimaru) [reply .github/workflows/ci.yml]: I've took your suggestion in #81 (co-authored by you).

Thanks for your suggestion 👍
